contents/blog/running-content-at-posthog.md

```diff
@@ -1,80 +1,112 @@
 ---
-date: 2021-08-06
-title: Content marketing strategy for devtool companies - How we do it at PostHog
-rootPage: /blog
-sidebar: Blog
+title: 10 A/B testing 
+sidebar: Docs
 showTitle: true
-hideAnchor: true
-author:
-  - mo-shehu
-featuredImage: ../images/blog/running-content.png
-featuredImageType: full
-category: Inside PostHog
-tags:
-  - Marketing
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: todo
+tags: ['experimentation']
 ---
 
-## Why you need content
-Developers are a selective bunch when it comes to what type of content they consume, and clickbait-y “Ultimate Guide To X” posts won’t fly with them. Your content needs to be detailed, thoughtful, and relevant to their problems.
+Making a mistake when running an A/B test can have disastrous implications. At best, it means you've wasted weeks waiting for results that are not useful. At worst, it can lead you to making the wrong product decisions.
```

Comment:
@Lior539 FYI, looks like you may have inadvertently saved some changes to another article.

Reply:
If you can't fix manually, you can always create a new branch and open a new PR.